core/src/main/java/com/opensymphony/xwork2/interceptor/annotations/AnnotationParameterFilterInterceptor.java

```diff
@@ -46,7 +46,10 @@
  * </p>
  *
  * @author martin.gilday
+ * @deprecated since 6.6.0, integrated into {@link ParametersInterceptor} with {@link StrutsParameter} using
```

Comment:
Do we have a task to remove this interceptor in Struts 7?

Reply:
I've just updated the description of [WW-5411](https://issues.apache.org/jira/browse/WW-5411) - if you'd prefer a separate card I can create one too

Reply:
That's fine, just to not forget about this :)